FIXED:(davmail with isync<1.3.1): https://sourceforge.net/p/isync/bugs/38/?limit=25
USAGE: $ loc_inst ./isync-git.pkgbuild

diff --git i/src/drv_imap.c w/src/drv_imap.c
index 05afa09..4fbc43e 100644
--- i/src/drv_imap.c
+++ w/src/drv_imap.c
@@ -2922,7 +2922,7 @@ imap_find_new_msgs_p2( imap_store_t *ctx, imap_cmd_t *gcmd, int response )
 	cmd->uid = cmdp->uid;
 	cmd->gen.param.lastuid = 1;
 	imap_exec( ctx, &cmd->gen, imap_find_new_msgs_p3,
-	           "UID FETCH *:* (UID)" );
+	           "UID FETCH * (UID)" );
 }
 
 static void
